docs/conf.py

```diff
@@ -63,7 +63,7 @@ def setup(sphinx):
 
 # General information about the project.
 project = 'Solidity'
-project_copyright = '2016-2021, Ethereum'
+project_copyright = '2016-2023, The solidity Authors'
```

Comment:
```suggestion
project_copyright = '2016-2023, The Solidity Authors'
```

Reply:
Wait, so who own the copyright, us, or EF?. Also, I think just `Solidity Authors` sounds better than `The Solidity Authors`

Reply:
Well technically, it's the Ethereum Foundation.

Reply:
So should it be `'2016-2023, Ethereum Foundation'`?

Reply:
The EF doesn't own copyright of all the various contributions to the docs we had. I don't think there is anything really accurate to put here - what's the implications of not having anything there, resp. displaying the license instead (isn't that more relevant for any practical purposes anyways?)

Reply:
If we want anything there at all, the circular references to the authors as whoever write it is probably as accurate as we can get :-)

Reply:
We could write ``2016-2023, licensed under GPL-3.0`` (and if we can add a link in the footer add a link to https://github.com/ethereum/solidity/blob/develop/LICENSE.txt). Honestly I think the reason why this is there in the first place is probably just because it was a field in the readthedocs config and not because we specifically _wanted_ a copyright there. The ``Ethereum`` copyright makes no sense in any case.